src/classes/PMT_PaymentWizard_CTRL.cls

```diff
@@ -539,12 +554,32 @@ public with sharing class PMT_PaymentWizard_CTRL {
 
     /*******************************************************************************************************
     * @description Returns true if the user has permissions to access, create and delete npe01__oppPayment__c
+    * and all relevant fields are both readable and modifiable
     * @return Boolean
     */
     private Boolean hasPaymentAccess() {
-        Schema.DescribeSObjectResult paymentDescribe = npe01__oppPayment__c.sObjectType.getDescribe();
+        Boolean accessResult = false;
+
+        SObjectType pmt = npe01__OppPayment__c.getSObjectType();
+        if (perms.canCreate(pmt) && UTIL_Permissions.canDelete(Schema.SObjectType.npe01__oppPayment__c.getName(), false)) {
+            Set<SObjectField> sObjectFields = new Set<SObjectField>{
+                npe01__oppPayment__c.fields.npe01__check_reference_number__c,
+                npe01__oppPayment__c.fields.npe01__custom_payment_field__c,
```

Comment:
This is a test field (which really shouldn't be in our package, but ....). Users do not require access to this field.